Cargo.toml

```diff
@@ -48,7 +48,7 @@ mediatype = "0.19"
 protobuf = { version = "3.3" }
 rand = { version = "0.8" }
 regex = { version = "1.10" }
-serde = { version = "1.0", features = ["derive"] }
+serde = { version = "1.0", features = ["derive"], optional = true }
```

Comment:
`serde_json` is also required by the cloudevent-sdk but is not really required by `uurivalidator::tests`. In fact, there is a test which reads in a JSON file that contains valid and invalid URIs ... IMHO this should be replaced by `#[test_case]`s to remove the (unnecessary) dependency on `serde_json`.